addons/im_livechat/static/src/models/public_livechat_window.js

```diff
@@ -59,9 +59,10 @@ registerModel({
                 $(this).on('click', self.messaging.publicLivechatGlobal.livechatButtonView.widget._onChatbotOptionClicked.bind(self.messaging.publicLivechatGlobal.livechatButtonView.widget));
             });
 
-            this.widget.$('.o_livechat_chatbot_main_restart').on('click',
-                this.messaging.publicLivechatGlobal.livechatButtonView.onChatbotRestartScript
-            );
+            this.widget.$('.o_livechat_chatbot_main_restart').on('click', (ev) => {
+                ev.stopPropagation(); // prevent fold behaviour
+                this.messaging.publicLivechatGlobal.livechatButtonView.onChatbotRestartScript();
```

Comment:
`onChatbotRestartScript` expects the event as an argument. Either pass it here or remove the parameter from the function definition.

Reply:
changes done (pass `ev` again)